Flavours/stellio-flavour/README.md

```diff
@@ -0,0 +1,49 @@
+# README
+
+This flavour exposes vThing information in [NGSI-LD](https://www.etsi.org/deliver/etsi_gs/CIM/001_099/009/01.02.02_60/gs_CIM009v010202p.pdf) format, through the [Stellio](https://github.com/stellio-hub/stellio-context-broker) Context Broker, Stellio is an NGSI-LD compliant context broker developed by EGM.
+
+NGSI-LD is an open API and Datamodel specification for context management published by ETSI.
+
+This flavour is for developers that want to use the NGSI-LD datamodel, which is based on the concept of Property Graphs, for representing IoT and context data, and the associated API, to provide and consume IoT data.
+
+
+## How To Run
+
+### Local Docker deployment
+
+First (as admin) add a "stellio flavour" to the system, based on the Docker image available on the fed4iot dockerhub, pointing to it with argument -i. Use the VirIoT CLI as admin and run the following command  (use "f4i.py add-flavour --help" for CLI parameters).
+
+```bash  
+python3 f4i.py add-flavour -f ngsild-stellio-f -i fed4iot/ngsild-stellio-f:2.2 -d "silo with a Stellio NGSI-LD broker" -s ""
```

Comment:
where does this 2.2 version come from?

Reply:
i followed the same pattern as in the other flavours
fed4iot/broken_name-f:2.2

Reply:
@LucaPetrucci what version can we choose for the fed4iot/ngsild-stellio-f image ?

Reply:
I suggest to build the docker image twice, one with tag 3.0 and the other one with tag latest.